lib/src/main/java/ca/jbrains/pos/PointOfSale.java

```diff
@@ -7,20 +7,20 @@
 
 public class PointOfSale {
     public static void main(String[] args) {
-        parseInput(new InputStreamReader(System.in)).forEachOrdered(
-                line -> displayToConsole(displaySellOneItem(line, (ignored) -> "::a hardcoded response for every barcode::"))
+        readBarcodeFrom(new InputStreamReader(System.in)).forEachOrdered(
+                barcodeInput -> displayToConsole(findProductPrice(barcodeInput, (ignored) -> "::a hardcoded response for every barcode::"))
```

Comment:
I think barcodeInput is better name than line

Reply:
What kind of "better" in this case?